app/models/concerns/wakeable/has_many.rb

```diff
@@ -8,6 +8,11 @@ module HasMany
       attr_accessor :has_many_label, :has_many_path
     end
 
+    def raw_aggregate_pageview_count
+      wakes_resources.reload
+      wakes_resources.sum("COALESCE((wakes_resources.document ->> 'pageview_count')::int, 0)")
```

Comment:
@joeosburn I am actually uncertain if the coalesce is strictly necessary here. I would rather not set a default value of 0, because nil is actually meaningfully different from 0 in this context. 0 would mean that there are actually 0 pageviews. nil would mean that we don't have data. And that distinction is carried through to the frontend.... admin should display 0 if 0, and "-" if nil.